lib/ansible/cli/galaxy.py

```diff
@@ -50,50 +50,14 @@
 
 class GalaxyCLI(CLI):
 
-    available_commands = {
-        "delete":    "remove a role from Galaxy",
-        "import":    "add a role contained in a GitHub repo to Galaxy",
-        "info":      "display details about a particular role",
-        "init":      "create a role directory structure in your roles path",
-        "install":   "download a role into your roles path",
-        "list":      "enumerate roles found in your roles path",
-        "login":     "authenticate with Galaxy API and store the token",
-        "remove":    "delete a role from your roles path",
-        "search":    "query the Galaxy API",
-        "setup":     "add a TravisCI integration to Galaxy",
-    } 
-
     SKIP_INFO_KEYS = ("name", "description", "readme_html", "related", "summary_fields", "average_aw_composite", "average_aw_score", "url" )
+    VALID_ACTIONS = ("delete","import","info","init","install","list","login","remove","search","setup")
```

Comment:
Spacing between the tuple items seems to be the recommendation in the CODING_GUIDELINES.md